SteamList/Controllers/GamesViewController.swift

```diff
@@ -53,8 +51,26 @@ extension GamesViewController: UITableViewDataSource {
     }
     
     
+     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+         navigate(tableView, indexPath)
+    }
+    
+    func navigate(_ tableView: UITableView,  _ indexPath: IndexPath) {
```

Comment:
The method name could be changed to something specific, currently it is too generic I would say

Reply:
sure